main.go

```diff
@@ -147,3 +138,48 @@ func main() {
 	// TODO: Missing clean shutdown and database disconnect.
 	log.Fatal(server.ListenAndServe(4000))
 }
+
+// loadDBCredentials creates a new db connection based on credentials found in
+// the environment variables.
+func loadDBCredentials() (string, options.Credential, error) {
+	var creds options.Credential
+	var ok bool
+	if creds.Username, ok = os.LookupEnv("SKYNET_DB_USER"); !ok {
+		return "", options.Credential{}, errors.New("missing env var SKYNET_DB_USER")
+	}
+	if creds.Password, ok = os.LookupEnv("SKYNET_DB_PASS"); !ok {
+		return "", options.Credential{}, errors.New("missing env var SKYNET_DB_PASS")
+	}
+	var host, port string
+	if host, ok = os.LookupEnv("SKYNET_DB_HOST"); !ok {
+		return "", options.Credential{}, errors.New("missing env var SKYNET_DB_HOST")
+	}
+	if port, ok = os.LookupEnv("SKYNET_DB_PORT"); !ok {
+		return "", options.Credential{}, errors.New("missing env var SKYNET_DB_PORT")
+	}
+	return fmt.Sprintf("mongodb://%v:%v", host, port), creds, nil
+}
+
+// loadPortalURLs returns a slice of portal urls, configured in the environment
+// under the key BLOCKER_SYNC_PORTALS. The blocker will keep in sync the
+// blocklist from these portals with the local skyd instance.
+func loadPortalURLs() (portalURLs []string) {
+	portalURLStr := os.Getenv("BLOCKER_PORTALS_SYNC")
+	for _, portalURL := range strings.Split(portalURLStr, ",") {
+		portalURL = sanitizePortalURL(portalURL)
+		if portalURL != "" {
+			portalURLs = append(portalURLs, portalURL)
+		}
+	}
+	return
+}
+
+// sanitizePortalURL is a helper function that sanitizes the given input portal
+// URL, stripping away trailing slashes and ensuring it's prefixed with https.
+func sanitizePortalURL(portalURL string) string {
+	portalURL = strings.TrimSpace(portalURL)
+	portalURL = strings.TrimPrefix(portalURL, "https://")
```

Comment:
how so? you mean because we're re-adding the prefix later? I don't mind that personally, it's straightforward:

It's 
```go
       portalURL = strings.TrimSpace(portalURL)
	portalURL = strings.TrimPrefix(portalURL, "https://")
	portalURL = strings.TrimPrefix(portalURL, "http://")
	portalURL = strings.TrimSuffix(portalURL, "/")
	return fmt.Sprintf("https://%s", portalURL)
```
vs
```go
       portalURL = strings.TrimSpace(portalURL)
	portalURL = strings.TrimSuffix(portalURL, "/")
	if strings.HasPrefix(portalURL, "https://") {
		return portalURL
	}
	portalURL = strings.TrimPrefix(portalURL, "http://")
	return fmt.Sprintf("https://%s", portalURL)
```

I'd take the first version any day, but I'll update I don't really care.